edge-middleware/bot-protection-datadome/.env.example

```diff
@@ -1,2 +1,4 @@
-NEXT_PUBLIC_DATADOME_CLIENT_KEY =
-DATADOME_SERVER_KEY =
\ No newline at end of file
+DATADOME_SERVER_SIDE_KEY =
+NEXT_PUBLIC_DATADOME_CLIENT_SIDE_KEY =
+# DATADOME_ENDPOINT = 
```

Comment:
Should these be here?

Reply:
Ok, I get your point, I'll let only the required keys.